auth_jwt/models/auth_jwt_validator.py

```diff
@@ -97,11 +96,25 @@ class AuthJwtValidator(models.Model):
     cookie_secure = fields.Boolean(
         default=True, help="Set to false only for development without https."
     )
+    renew_cookie_on_response = fields.Boolean(
+        help="Renew the cookie in every response", default=True
+    )
 
     _sql_constraints = [
         ("name_uniq", "unique(name)", "JWT validator names must be unique !"),
     ]
 
+    @api.constrains("signature_type", "renew_cookie_on_response")
+    def _check_signature_type_with_cookie_renew(self):
+        for rec in self:
+            if rec.renew_cookie_on_response and rec.signature_type != "secret":
+                raise ValidationError(
+                    _(
+                        "The sginature type should be secret if the cookie is "
+                        "renewed on responses"
+                    )
+                )
```

Comment:
I don't think we should error in this case. Creating a http-only secure cookie once the client has been authenticated with the bearer token. If anything it makes the life of front-end developers easier (and more secure) as they don't need to worry with securely storing the JWT token, as the browser handles the cookie securely for them. I do agree with the caveat regarding session invalidation.